### ISSUE TITLE
Auth token is not refreshed

### ISSUE THREAD

**Issue by bijanmj** [directus/app#1982]
The app gets disconnected every 5 minutes and I have to log back in again. The token refresh endpoint is never hit.
Here's what I have in sessionStorage:
`{"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MSwiZXhwIjoxNTY2NTA4NDk3LCJ0eXBlIjoiYXV0aCIsImtleSI6Ijx0eXBlLWEtcHVibGljLWF1dGhlbnRpY2F0aW9uLWtleS1zdHJpbmc-IiwicHJvamVjdCI6Il8ifQ.2SKSsDZ8WszHVZ92bcBzsig0WTaeu8weVsFXkepdZa4","url":"http://api.admin.dbrs.biz:5000","project":"_","localExp":1566509408586}`

**Comment by rijkvanzanten** [comment]
What version of the app? How did you log in (SSO?)? Do you have any extensions that might interfere? What browser?

We could really use some more info cause there's no way to reproduce this issue based on the description you've provided..

**Comment by bijanmj** [comment]
I installed the latest version of the app (7.9.0), signed in via username/password through the Apps's signin box. I tested it on 3 different browsers with no plugins: 
- Chrome (75.0.3770.142)
- Firefox (68.0.2)
- Safari (12.1.2)

All browsers behave exactly the same, no token refresh requests. The only periodic request I get, is the server ping. 
I'm hosting the app on S3, not sure if that could be a potential cause of this issue.

**Comment by theharshin** [comment]
I can reproduce the issue and the cause is the `null` payload. The `api.loggedIn` check returns `false` if the payload in SDK is `null`. The payload is generated with the help of token. The initial token when logged in looks like this:

`eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MSwiZXhwIjoxNTY3MDczODQ0LCJ0eXBlIjoiYXV0aCIsImtleSI6Ijx0eXBlLWEtcHVibGljLWF1dGhlbnRpY2F0aW9uLWtleS1zdHJpbmc-IiwicHJvamVjdCI6Il8ifQ.rPjDJjTNVoqACyEn2FFu0_TM1hZpBBUkoxZglE4LsSk`

While the refresh token endpoint returns this token:
`eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.Im51bGwi.KIGRv7EhA9410Tk-B6PaAi_t2_1pgBE5DacNRrEA-zQ`

As you can see the formats are different and hence `getPayload` method fails to generate payload.

@bjgajjar Any idea why the token formats are different?

**Comment by bjgajjar** [comment]
Yes, [2FA PR](https://github.com/directus/api/commit/8bfd3f35d0a4db7722e15cec14595648d14bce3a#diff-1ad2976857a8221713ce220258c409e2) is the reason behind a different format in a refresh token.

Let's move this to `API`
